### PR TITLE
W-14624620 RPA Manager release notes 2023-12/1.

### PR DESCRIPTION
**Pull request by jhammann72**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,45 @@
...
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[december-11-2023]]
+== December 11, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* RPA Manager now implements Anypoint Audit Logging, which enables you to see a queryable history of actions performed within Anypoint Platform.
+* The *Usage Summary* is now grouped by the process' category.
```
Comment (sebermuley) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: by the processes' categories
OR
by process category

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,45 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[december-11-2023]]
+== December 11, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* RPA Manager now implements Anypoint Audit Logging, which enables you to see a queryable history of actions performed within Anypoint Platform.
```
Comment (sebermuley) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: Maybe:
actions performed with RPA objects
?

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Upcoming Process Changes* view no longer shows an error when selecting the table header *Last edited*. | W-14293208
+| *Process Run Results* no longer show *Endtime* and *Duration* if the respective process is in status _running_. | W-14311008
+| You can now download RPA Bot logs in the *Bot Management* when the RPA Bot is in status _out of service time_. | W-14378109
+| You can now use the same name for global variables and credentials in different organizations. Names are unique within the same organization. | W-14378876
+| The close button of the videos player in the home page no longer dissappears after resizing the browser's window. | W-14387182
+| You can now create a global variable of type integer with a negative number value. | W-13460628
```
Comment (sebermuley) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: with a negative value

**Review by sebermuley — DISMISSED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 4]
```diff
@@ -2,6 +2,45 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[december-11-2023]]
+== December 11, 2023
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
[[december-12-2023]]
== December 12, 2023
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 5]
```diff
@@ -2,6 +2,45 @@
...
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[december-12-2023]]
+== December 12, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+* RPA Manager now implements Anypoint Audit Logging, which enables you to see a queryable history of actions performed with RPA objects.
+* The *Usage Summary* is now grouped the process category.
```
Comment (dmerlob) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
* The *Usage Summary* is now grouped in the process category.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 6]
```diff
@@ -2,6 +2,45 @@
...
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| The *Upcoming Process Changes* view no longer shows an error when selecting the table header *Last edited*. | W-14293208
+| *Process Run Results* no longer show *Endtime* and *Duration* if the respective process is in status _running_. | W-14311008
+| You can now download RPA Bot logs in the *Bot Management* when the RPA Bot is in status _out of service time_. | W-14378109
+| You can now use the same name for global variables and credentials in different organizations. Names are unique within the same organization. | W-14378876
+| The close button of the videos player in the home page no longer dissappears after resizing the browser's window. | W-14387182
+| You can now create a global variable of type integer with a negative value. | W-13460628
+| The dashboard widget *Business Process Performance* now aggregates successful and failed process executions. | W-14438065
```
Comment (dmerlob) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:27]:
```suggestion
| The dashboard widget *Business Process Performance* now aggregates successful and failed process executions. | W-14438065
```
is *failed* correct?

**Review by dmerlob — DISMISSED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc (hunk 5 above)
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
* The *Usage Summary* is now grouped by process category.
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc (hunk 6 above)
Comment (Cristian-Venticinque) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:27]: Yes, to make it more clear maybe I can reword to: _now aggregates **both** successful and failed process executions._

Comment (dmerlob) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:27]: **Both** works fine!

**Review by dmerlob — APPROVED** [review]

**Pull request merged**